status-whisper-usage-spec.md

```diff
@@ -52,6 +53,23 @@ encryption properties to support asynchronous chat.
 * *Offline message*: an archived envelope
 * *Envelope*: encrypted message with metadata like topic and Time-To-Live
 
+## Whisper packets
+
+| Packet Name | Code | EIP-627 | References |
+| --- | --: | --- | --- |
+| Status | 0 | ✔ | [Handshake](#handshake) |
+| Messages | 1 | ✔ | [EIP-627](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-627.md) |
+| PoW Requirement | 2 | ✔ | [EIP-627](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-627.md) |
+| Bloom Filter | 3 | ✔ | [EIP-627](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-627.md) |
+| Batch Ack | 11 | 𝘅 | TODO |
```

Comment:
When are these behaviors going to be described? Currently it is rather opaque, especially if this blocks nim-eth mailserver client capability implementation.

Reply:
@oskarth https://github.com/status-im/specs/pull/74